2024/2024-ssl-teams-vs-teams-passing-challenge.adoc

```diff
@@ -0,0 +1,94 @@
+females.
+
+== Goal
+
+A key distinguishing characteristic of the Small Size League is its tight integration of software and hardware, which is crucial to create a dynamic, collaborative soccer match. 
+While this goal was pursued in last year's SSL Chip Pass Challenge, the goal of this year's challenge is to utilize last year's work and adapt it to a more realistic situation.
+
+Therefore this year two teams will try to play as many passes as possible while competing against each other. To achieve this, the ruleset used last year has been adapted to the new situation.
+
+Although the teams, which have participated in one of the previous passing challenges, might have a slight advantage, we would like to encourage all teams to participate in this technical challenge.
+
+== Procedure
+
+The technical challenge will be held on a Div B field with a maximum of 4 robots per team.
+
+To level the playing field and to remove the impact of the performance of the opponent team on a teams score, each team will play two games against different contestants of the technical challenge.
+
+Each game will have the following procedure:
+
+* To reduce the preparation time, the team colors for each team will be determined by the TC/OC and announced togheter with the schedule of the technical challenge.
+* To start the game, a kick-off command for the yellow team will be issued from the game controller. For the start the robots can be positioned in any way preferred, as long as the positioning does not violate the usual ruleset for a kick-off.
+* After *2 minutes of game time* a stop command will be issued and the first half of the game is over.
+* The half time break for this challenge will be only 60 seconds, since it should be mainly used to set up the robots at their desired position again and to exchange faulty robots. 
+* To start the second half a kick-off command for the blue team will be issued.
+* After another *2 minutes of game time* a stop command will be issued and the game is over. 
+
+=== Special Rule Set during the Technical Challenge
+
+For the technical challenge a Div B field will be used. If the ball leaves the field via any of the field lines, the game will be halted and the team that did not last touch the ball gets awarded a kick-off.
+
+During the games of the technical challenge *only* the following fouls from the ssl rulebook apply: 
+
+* Pushing 
+* Ball Holding 
+* Tipping over or dropping parts 
+* Boundary Crossing 
+* Excessive Dribbling 
+* Ball Speed 
+* Crashing 
+* Defender too close to ball 
+* Robot Stop Speed 
+
+If there is no progress in the game for 10 seconds while both teams are allowed to manipulate the ball, the game is stopped and the team that got the first kick-off of this halftime will get a kick off.
+
+=== Pass Definitions
+
+Teams are free in how they perform the passes. Definitions of a pass in general and additional constraints for valid passes that are considered in the scoring are given in this section.
```

Comment:
The sentence is not correct: The valid passes are described in the next section ;)

Reply:
Should be fixed with the next commit as well